Libraries/LibJS/Bytecode/Op.h

```diff
@@ -2373,6 +2373,17 @@ class ThrowIfTDZ final : public Instruction {
     Operand m_src;
 };
 
+class ThrowInvalidAssignToConst final : public Instruction {
```

Comment:
issue: Since this is an unconditional throw, IsTerminator needs setting to true
suggestion: Copy the line from Throw